spec/requests/admin/ccms_connectivity_tests_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe Admin::CcmsConnectivityTestsController, type: :request do
+  let(:count) { 3 }
+  let!(:legal_aid_applications) { create_list :legal_aid_application, count, :with_applicant }
+  let(:admin_user) { create :admin_user }
+
+  before { sign_in admin_user }
+
+  describe 'GET /admin/ccms_connectivity_tests/:id' do
+    let(:response_stub) do
+      <<~XML
+        <?xml version='1.0' encoding='UTF-8'?>
+        <note>
+          <body>response stub</body>
+        </note>
+      XML
+    end
+
+    let(:requestor) do
+      instance_double(CCMS::ReferenceDataRequestor, call: response_stub)
+    end
+
+    subject { get admin_ccms_connectivity_test_path(legal_aid_applications.first.id) }
+
+    before do
+      allow(CCMS::ReferenceDataRequestor).to receive(:new).and_return(
+        instance_double(CCMS::ReferenceDataRequestor, call: response_stub)
```

Comment:
should you use `requestor` defined on line 20 here?

Reply:
Wops, my bad. Originally defined the `requestor` object, but later decided to embed it in the `before` stub in order to make it more compact and easier to understand (as would not use it outside this single case). But seems I forgot to remove the `requestor` definition.

Going to remove the `let(:requestor)` instead.